node/cmd/spy/spy.go

```diff
@@ -304,6 +311,17 @@ func runSpy(cmd *cobra.Command, args []string) {
 	// Guardian set state managed by processor
 	gst := common.NewGuardianSetState(nil)
 
+	// VAA verifier (optional)
+	if *ethRPC != "" {
+		if *ethContract == "" {
+			logger.Fatal(`If "--ethRPC" is specified, "--ethContract" must also be specified`)
+		}
+		vaaVerifier = NewVaaVerifier(logger, *ethRPC, *ethContract)
+		if err := vaaVerifier.GetInitialGuardianSet(); err != nil {
+			logger.Fatal(`Failed to read initial guardian set for VAA verification`, zap.Error(err))
```

Comment:
since we don't need to log double quotes `"` in the string here should we standardize this to double quotes instead of backticks?

Reply:
Is there a downside to using back ticks? Because I'm actually thinking I should start always using back ticks so I can double quote things in the message without thinking about it. . .

Reply:
I don't think so! In fact it is more readable when you have double quotes in the string. Was just nitpicking because I thought double quote was the standard, since most of the code is using double quotes.